### ISSUE TITLE
Pronunciation FAQ

### ISSUE THREAD

**Issue by swrobel** [shadcn-ui/ui#2541]
Just came across this project and it seems odd to me that there isn't a pronunciation guide in the FAQs.

**Comment by shadcn** [comment]
This issue has been automatically closed because it received no activity for a while. If you think it was closed by accident, please leave a comment. Thank you.

**Comment by danedwardsdeveloper** [comment]
I have to agree here. It's off putting if you can't pronounce something because it makes talking about it a bit embarrassing. Do you remember Chipotle had an entire campaign about pronouncing it properly?

Vite has addressed their pronunciation in the very first paragraph or their docs!

<img width="549" alt="Image" src="https://github.com/user-attachments/assets/9612afcd-8783-4abc-975c-57d9425345f4" />

**Comment by Sax388** [comment]
I still couldn't find it here: https://ui.shadcn.com/docs/registry/faq.

I know that it's `/ʃæd siː ɛn/` (according to a credible source on a not so credible platform: https://x.com/shadcn/status/1647359389056073731?lang=en) so let's put it here as well for other's stumbling across it.

**Comment by fr34kyn01535** [comment]
If true, the pronunciation of "chadcn" is close to the German "Schätzchen", a diminutive of "Schatz" (‘treasure’). It translates to ‘sweetheart,’ and colloquially can mean ‘babe,’ sometimes with a slightly derogatory tone.

**Comment by shadcn** [comment]
haha I'll get to it ;)
